src/components/Editable.tsx

```diff
@@ -443,6 +445,16 @@ const Editable = ({ disabled, isEditing, thoughtsRanked, contextChain, cursorOff
     }
   }
 
+  /**
+   * Prevents onKeyDownAction call for shift, alt or ctrl keys.
+   */
+  const onKeyDown = (e: React.KeyboardEvent) => {
+    if ((e.shiftKey || e.altKey || e.ctrlKey || e.metaKey) && MODIFIER_KEYS.find(keyName => keyName === e.key)) {
+      return
+    }
```

Comment:
Glad you created a data structure for `MODIFIER_KEYS`!

Rather than store `MODIFIER_KEYS` as an array, store it as an object to allow O(1) lookup by key. This will also mitigate the explicit check for `e.shiftKey || e.altKey || e.ctrlKey || e.metaKey`.